### PR TITLE
Support container images with mesos containerizer 

### PR DESCRIPTION
**Pull request by kamaradclimber**
This PR introduces support to launch jenkins slave with mesos containerizer within a docker image.
This allows to benefit from mesos containerizer while getting filesystem isolation.

This is my first jenkins plugin patch and I would really appreciate feedback/advices (on jelly files especially).

PR also build against latest mesos version although it is not strictly necessary (I can remove this commit if needed)

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/mesos/MesosSlaveInfo/ContainerInfo/config.jelly [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
     <f:entry title="${%Container Type}" field="type">
-        <f:radioBlock name="type" title="${%Docker}" value="DOCKER" inline="true" checked="true">
-            <f:entry title="${%Docker Image}" field="dockerImage" description="If using Docker, specify the docker image.">
+            <f:entry title="${%Docker Image}" field="dockerImage" description="Specify the docker image.">
                 <f:textbox clazz="required"/>
```
Comment (vinodkone) [inline src/main/resources/org/jenkinsci/plugins/mesos/MesosSlaveInfo/ContainerInfo/config.jelly]: Can you paste before and after screenshots of how this looks in Jenkins configure page?

File: pom.xml [hunk 2]
```diff
@@ -95,8 +95,8 @@
     <jenkins.version>2.7.3</jenkins.version>
     <java.level>8</java.level>
     <findbugs.failOnError>false</findbugs.failOnError>
-    <mesos.version>1.0.0</mesos.version>
-    <protobuf.version>2.6.1</protobuf.version>
+    <mesos.version>1.7.0</mesos.version>
```
Comment (vinodkone) [inline pom.xml]: Lets do this in a separate PR if this change doesn't require mesos version or protobuf version to be bumped.

File: README.md [hunk 3]
```diff
@@ -76,6 +76,8 @@ By default, the Jenkins slaves are run in the default Mesos container. To run th
 
 	2) "Use External Containerizer" : Select this option if Mesos slave(s) are configured with "--containerizers=external".
```
Comment (vinodkone) [inline README.md]: I would just rephrase this instead

2) "Use Mesos Containerizer": Select this option if Mesos slave(s) are configured with "--containerizers=mesos"

**Comment by kamaradclimber** [comment]
Closing since UCR support should be merged by mesosphere
